{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -121,15 +141,15 @@ matrix:
           env: NUMPY_VERSION=prerelease
                EVENT_TYPE='pull_request push cron'
 
-        # Do a PEP8 test with pycodestyle
+        # Do a PEP8 test with flake8
         - os: linux
-          env: MAIN_CMD='pycodestyle {{ cookiecutter.module_name }} --count' SETUP_CMD=''
+          env: MAIN_CMD='flake8 {{ cookiecutter.module_name }} --count $FLAKE8_OPT' SETUP_CMD=''
```

Comment:
I like  __--show-source__ and/or __--statistics__ more than __--count__.

Reply:
I'll leave this one for the maintainers to decide...

Reply:
`--show-source` can be an overkill but arguably very helpful in CI for people without having a linter set up in their editor. 
Also ``--statistics`` is indeed a good to have (can be done along with ``--count``)